likeminds-chat-reactnative-integration/ChatSX/components/Swipeable/index.tsx

```diff
@@ -76,7 +76,7 @@ const Swipeable = ({
     event: GestureUpdateEvent<PanGestureHandlerEventPayload>
   ) => {
     "worklet";
-    if (event.translationX >= 75) {
+    if (event.translationX >= 90) {
```

Comment:
why do you need to change the animation code?

Reply:
reverted this change as this was just to increase the scroll along x to trigger reply